charts/Geomap.stories.js

```diff
@@ -16,16 +16,111 @@ GettingStarted.args = {
   data: []
 };
 
+export const ChangingCoordinatePointSizes = Template.bind({});
+ChangingCoordinatePointSizes.args = {
+  data: "https://d3plus.org/data/city_coords.json",
+    groupBy: "slug",
+    colorScale: "dma_code",
+    label: funcify(function(d) {
+        return d.city + ", " + d.region;
+      }, `function(d) {return d.city + ", " + d.region}`),
```

Comment:
I would rather show ES6 fat arrow functions and string templates where we can, what do you think? So this would become:

```js
d => `${d.city}, ${d.region}`,
```

Reply:
I really like the idea! They are already up to date on args and stories files (where I was able to) in the last commit 😏.